wgengine/router/router_passthrough.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package router
+
+import (
+	"github.com/tailscale/wireguard-go/device"
+	"github.com/tailscale/wireguard-go/tun"
+	"tailscale.com/types/logger"
+)
+
+// SetRoutesFunc applies the given router settings to the OS network stack.
+// cfg is guaranteed to be non-nil.
+var SetRoutesFunc func(cfg *Config) error
+
+type passthroughRouter struct{}
```

Comment:
then we can put the func in this.

also, godoc on this type please, even if minimal.